packages/web-components/fast-foundation/src/anchor/anchor.ts

```diff
@@ -87,7 +87,7 @@ export class Anchor extends FASTElement {
  *
  * @public
  */
-export class DelegatesARIALink extends ARIAGlobalStatesAndProperties {
+export class DelegatesARIALink {
```

Comment:
I think since you're mixing `ARIAGlobalStatesAndProperties` into `DelegatesARIALink` you'll want to merge the declarations as well:

```ts
interface DelegatesARIALink extends ARIAGlobalStatesAndProperties {}
```

As-is the runtime type is inconsistent w/ the TS types.

Reply:
Good call. Looks like some of the tests are failing related to this. I'll get it fixed shortly.